frontend/pages/index.tsx

```diff
@@ -7,7 +7,8 @@ const Home: NextPage = () => {
       <Head>
         <title>Manipal OSF</title>
       </Head>
-      <div className='m-auto grid place-self-center text-center dark:text-white'>
+      <div className='grid m-auto gap-5 text-center place-self-center text-secondary dark:text-secondary-dark'>
+        <h2 className='duration-300 text-left italic hover:-translate-y-1 text-accent dark:text-accent-dark'>Experience</h2>
```

Comment:
The `Experience` is not aligned to center in small viewports.

![image](https://user-images.githubusercontent.com/72333092/219953156-d6b7af01-e73e-47b3-90e2-68ee97b5f05a.png)

Perhaps align all `div` contents to the left and center the `div`

Reply:
Fixed using a breakpoint to reduce the size of Manipal OSF